packages/core/src/bus/Event.ts

```diff
@@ -0,0 +1,30 @@
+import { EventDispatcher } from './EventDispatcher';
+import { getTypeName } from '../utils';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
Avoid aliases if it is possible

```suggestion
import { v4 } from 'uuid';
```